Jenkinsfile

```diff
@@ -92,18 +92,23 @@ pipeline {
                         }
                     }
                     steps {
+                        echo 'Cleaning up old build artifacts'
+                        sh 'rm -rf ./target/debug/deps/libamethyst* || true'
```

Comment:
This also doesn't work reliably, because as soon as we have a new patch version in any of our dependencies, `mdbook` will fail again.

Reply:
yeap, just realized; I'm crafting a bash script, which isn't great, but I think it's worth testing the book